llama_index/callbacks/base.py

```diff
@@ -49,6 +63,9 @@ class CallbackManager(BaseCallbackHandler, ABC):
     def __init__(self, handlers: List[BaseCallbackHandler]):
         """Initialize the manager with a list of handlers."""
         self.handlers = handlers
+        self._trace_map: Dict[str, List[str]] = defaultdict(list)

```

Comment:
I don't think a reset works for the init (since the trace_id/running_id is set/unset at a different time compared to the trace_map and trace_stack, but I can add a reset for those two